RELEASE_PROCESS.md

```diff
@@ -0,0 +1,42 @@
+## Release process
+
+### Target audience
+
+This document is aimed toward established contributors the project.
+
+If you're a user looking to install dirty_cat, please refer to
+the instructions in the [README](README.rst) !
+
+### Process
+
+> Going further, we assume you have write-access to both the repository and 
+> the PyPI project page.
+
+To release a new version of dirty_cat,
+here are the main steps and appropriate resources:
+
+1. Update `dirty_cat/CHANGES.rst`. It should be updated at each PR,
+   but double-checking before the release is good practice.
+2. Update `dirty_cat/dirty_cat/VERSION.txt` with the new version number
+3. Update `dirty_cat/setup.py`
+4. You have to commit the changes with a new tag: 
+   the version you're going to push (e.g. `0.1`)
```

Comment:
Maybe adding the explicit commands here :
git commit -m "prepare for v1.0.0 release"
git tag v1.0.0
git push

Reply:
Otherwise, everything is good.